docs/tutorials/FlowTutorial.md

```diff
@@ -817,7 +829,7 @@ The `gcd` design synthesis results for area and speed optimizations are shown be
 |-----------------------|--------------------------------------|--------------------------------------|
 | `Number of wires`     | 224                                  | 224                                  |
 | `Number of wire bits` | 270                                  | 270                                  |
-| `Number of cells`     | 234                                  | 234                                  |
+[O| `Number of cells`     | 234                                  | 234                                  |
```

Comment:
Something weird here.

Reply:
Thanks, will fix this with other doc fixes later on.